themes/bootstrap3/templates/search/history-table.phtml

```diff
@@ -43,7 +43,7 @@
             <?php endif; ?>
           <?php endforeach; ?>
         </td>
-        <td class="history_results"><?=$this->escapeHtml($this->localizedNumber($info->getResultTotal()))?></td>
+        <td class="history_results" data-label="<?=$this->transEscAttr('history_results')?>"><?=$this->escapeHtml($this->localizedNumber($info->getResultTotal()))?></td>
```

Comment:
I think we need a data-label on the search-schedule-header td below to resolve @sturkel89's reported problem.

Is it possible that we should also label the history_actions td further down (the container for the save/delete buttons), or is that okay without a label? (I think it seems fairly self-explanatory without the label, but just making sure there's not a reason to add something).

Reply:
I think actions is okay without a label. Next, I'll check @sturkel89 suggested enhancements.